src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/NoWhitespaceBeforeCheck.java

```diff
@@ -61,6 +61,23 @@
  * <pre>
  * &lt;module name=&quot;NoWhitespaceBefore&quot;/&gt;
  * </pre>
+ * <p>Example of violation:</p>
+ * <pre>
+ * int a = a ++ + a --; // '++' and '--' are preceded with whitespace
```

Comment:
`int a = a ++ + a --` 
This code is invalid and too complex. 
I think only one of these operators is enough.